File: Programming Basics/For Loop - Lab/10.OddEvenSum.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        int n = int.Parse(Console.ReadLine());
        double oddSum = 0;
        double evenSum = 0;

        for (int i = 1; i <= n; i++)
        {
            if (i % 2 == 0)
            {
                evenSum += double.Parse(Console.ReadLine());
            }
            else
            {
                oddSum += double.Parse(Console.ReadLine());
            }
        }
        if (evenSum == oddSum)
        {
            Console.WriteLine($"Yes\r\nSum = {evenSum}");
        }
        else
        {
            Console.WriteLine($"No\r\nDiff = {Math.Abs(evenSum - oddSum)}");
        }
    }
}

File: Programming Basics - Python/For Loop - Lab/10.OddEvenSum.py
n = int(input())

even_sum = 0
odd_sum = 0

for num in range(0, n):
    number = int(input())

    if num % 2 == 0:
        even_sum += number
    else:
        odd_sum += number

if even_sum == odd_sum:
    print("Yes")
    print(f"Sum = {even_sum}")
else:
    print("No")
    print(f"Diff = {abs(even_sum - odd_sum)}")